.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/dceara/ovn-heater/blob/main/CONTRIBUTING.md
+
+** Make sure all your commits include a signature generated with `git commit -s` **
+
+If this is a bug fix, make sure your description includes "Fixes #xxxx", or
+"Closes #xxxx"
```

Comment:
I was under the impression that GH wouldn't like the semicolons but I tried it out on a test repo and it will still automatically close the referenced issues.  I'll add the semicolons.